pkg/webhook.v2/setup.go

```diff
@@ -18,6 +18,15 @@ package webhookv2
 
 import ctrl "sigs.k8s.io/controller-runtime"
 
-func Setup(ctrl.Manager) (string, error) {
+func Setup(mgr ctrl.Manager) (string, error) {
+	if err := setupWebhookForClusterTrainingRuntime(mgr); err != nil {
+		return "ClusterTrainingRuntime", err
+	}
+	if err := setupWebhookForTrainingRuntime(mgr); err != nil {
+		return "TrainingRuntime", err
+	}
+	if err := setupWebhookForTrainJob(mgr); err != nil {
+		return "TranJob", err
```

Comment:
Can we move those Kind types to constants ?

```suggestion
		return "TrainJob", err
```

Reply:
Done.